extensions/wikia/WikiFactory/DeadWikis/maintenance.php

```diff
@@ -292,10 +290,17 @@ protected function getWikisList() {
 			"l.city_created < \"".wfTimestamp(TS_DB,strtotime(self::$FETCH_TIME_LIMIT))."\"",
 			"(v.city_flags IS NULL OR v.city_flags & " . WikisModel::FLAG_OFFICIAL . " = 0)",
 		);
-		if (!is_null($this->from))
-			$where[] = "l.city_id >= ".intval($this->from);
-		if (!is_null($this->to))
-			$where[] = "l.city_id <= ".intval($this->to);
+		if (!is_null($this->ids) && !empty($this->ids)) {
+			foreach ($this->ids as $id) {
+				$where[] = "l.city_id = ".intval($id);
+			}
```

Comment:
The `$db->select` call will construct this for you using an `IN` clause if you pass it the array, e.g.:
```php
$where["l.city_id"] = $this->ids
```